basedecorator.rb

```diff
@@ -0,0 +1,12 @@
+require './nameable'
+
+class Basedecorator < Nameable
```

Comment:
- [ ] [OPTIONAL] Ensuring and defining access (getter and setter methods) to `nameable` with the use of `attr_accessor` so that it can be accessible outside the class if needs be :wink:

Reply:
well noted thanks